versioned_docs/version-V3/guides/liquidity-mining/overview.md

```diff
@@ -0,0 +1,42 @@
+---
+id: liquidity-mining-overview
+title: Overview
+sidebar_position: 1
+---
+
+## Introduction
+
+As a DeFi project, token creator, or other interested party, you may be wondering what the best ways to _incentivize in-range liquidity provision_ on one or more Uniswap V3 pools are. This guide describes one particular incentivization scheme at a high level, as implemented in [uniswap-v3-staker](https://github.com/Uniswap/uniswap-v3-staker).
+
+## The Setting
+
+Let's start by defining some terms. First, we refer to programs which incentivize liquidity as `Incentive`s. `Incentive`s are characterized by the following parameters:
```

Comment:
the double incentives render funny :).  Is it actually variable name?
<img width="768" alt="Screen Shot 2021-08-03 at 7 37 31 PM" src="https://user-images.githubusercontent.com/5455577/128099571-d8092319-d7e0-439e-b11d-3c8dfcbc56ab.png">